app/models/legal_aid_application.rb

```diff
@@ -1,4 +1,7 @@
 class LegalAidApplication < ApplicationRecord
+
+  has_one :applicant
```

Comment:
minor cosmetic change, i think the before** methods should be at the top of a class